core/deployment/src/main/java/io/quarkus/deployment/pkg/ManifestConfig.java

```diff
@@ -12,4 +15,11 @@
     @ConfigItem(defaultValue = "true")
     public boolean addImplementationEntries;
 
+    /**
+     * Custom manifest sections to be added to the MANIFEST.MF file.
+     * An example of the user defined property:
+     * quarkus.package.manifest.manifest-sections={Section Name}:{Key1}={Value1},{Section Name}:{Key2}={Value2}
```

Comment:
@aloubyansky @stuartwdouglas are you OK with this format?

Reply:
I think this should be represented as a Map, rather than a fully de-typed format.